composer.json

```diff
@@ -9,11 +9,13 @@
             "email": "z.aliakseyeu@gmail.com"
         }
     ],
-     
+
     "require": {
-        "symfony/framework-bundle": "~2.2",
+        "symfony/framework-bundle": "~3.4",
```

Comment:
@Anatoliy2017 hi!

Do we have a good reason to require symfony version starting from 3.4?

I can see one change that requires symfony version upgrade - `FormType:getName` -> `FormType:getBlockPrefix`

This change was introduced in 2.8 symfony version according to their docs & history https://symfony.com/doc/2.8/form/create_custom_field_type.html#creating-a-template-for-the-field
https://github.com/symfony/symfony/pull/15079/files#diff-6c37a005ac1e594590b9625bcf91eddf97e9a4c61ea8c68bfef00834571b07c6R70

So technically we can support starting from 2.8 even it's deprecated now.
I think better to support lowest possible version.

Reply:
Since bundle not so popular and we have no automated test builds to check if all new sf versions are compatible, we can just assume it will work with any sf version higher that 2.8

I propose to set just  "symfony/framework-bundle": ">=2.8"